packages/theme-language-server-common/src/json/JSONLanguageService.ts

```diff
@@ -71,13 +70,8 @@ export class JSONLanguageService {
             },
           },
 
-          // Custom non-JSON schema completion & hover contributions
           contributions: [
-            new TranslationFileContributions(this.documentManager),
-            new SchemaTranslationContributions(
-              this.documentManager,
-              this.getDefaultSchemaTranslations,
-            ),
+            new JSONContributions(this.documentManager, this.getDefaultSchemaTranslations),
```

Comment:
Devs shouldn't need to worry about this anymore other than ferrying new dependencies in there.